app/src/main/res/layout/activity_orbot.xml

```diff
@@ -2,6 +2,7 @@
 <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_marginTop="140dp"
```

Comment:
A recent change like this 9440e723f29b9b282474b0a192f96811a588d12e caused a regression in the UI for lower resolution devices. This change fixed the 140dp margin by doing this: https://github.com/guardianproject/orbot-android/pull/1249/files

I think a similar fix like above is also required here.